### PR TITLE
Post styling

### PR DESCRIPTION
**Pull request by ocripps24**
Styling has been implemented for all current posts pages i.e. posts-index and posts-new. The styling is consistent across both pages and can be expanded into an edit page in the future.

### CODE REVIEW

**Review by Nandini0206 — APPROVED** [review]
CSS file is easy to understand, I like how the way everything is split into CSS for each page

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,17 +1,75 @@
...
-    data: { confirm: 'Are you sure?' } %>
-    <%end%>
-<% end %>
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+
+    <link href="https://fonts.googleapis.com/css?family=Raleway:400,700&display=swap" rel="stylesheet">
+    <title>Acebook - Posts</title>
```
Comment (shektor) [inline app/views/posts/index.html.erb:7]: To be updated to Bearbook

File: app/views/posts/new.html.erb [hunk 2]
```diff
@@ -1,6 +1,60 @@
...
-  <%= form.text_area :message %>
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
 
-  <%= form.submit "Submit" %>
-<% end %>
+    <link href="https://fonts.googleapis.com/css?family=Raleway:400,700&display=swap" rel="stylesheet">
+    <title>Acebook - Posts</title>
```
Comment (shektor) [inline app/views/posts/new.html.erb:7]: To be updated to Bearbook

**Review by shektor — APPROVED** [review]
Great CSS and the forms have been encapsulated correctly.

**Pull request merged**